specifyweb/businessrules/views.py

```diff
@@ -1 +1,316 @@
 # Create your views here.
+from .models import UniquenessRule
+import json
+
+from django import http
+from django.db import transaction
+from django.db.models import Q, Count
+from django.views.decorators.http import require_http_methods, require_POST
+
+from specifyweb.specify.views import login_maybe_required, openapi
+from specifyweb.specify import models
+from specifyweb.specify.models import datamodel
+from specifyweb.permissions.permissions import PermissionTarget, PermissionTargetAction, check_permission_targets
+
+from .uniqueness_rules import make_uniqueness_rule
+
+
+class SetUniqueRulePT(PermissionTarget):
+    resource = "/schemaconfig/uniquenessrules"
+    view = PermissionTargetAction()
```

Comment:
Can you call it `read` instead of `view`? More consistent with the naming style on the backend for such permissions.

Reply:
I initially used `read`, although it appears this permission was being set by default and additionally marked as always required when setting the permission. 
I will do some further looking into this and confirm that was the case.

Reply:
Using `read` forces that action to be marked as required. 

See https://github.com/specify/specify7/blob/fd09dafb06e36b6c687b8ace7179b2314fb8444e/specifyweb/frontend/js_src/lib/components/Security/Policy.tsx#L233

Currently, the permission denotes the ability to view the Uniqueness Rules from the Schema Configuration: hence why I secondarily opted for `view`. The user is not really 'reading' the rules (that is, fetch them from the backend): this should always happen and not have a permission associated with it.  

Which solution would be preferable? 
Modifying the permission code to allow `read`be not required (would probably be beneficial for other similar permissions in the future) or rename the permission? 
Or rather, should every user be able to at least see the uniqueness rules for any given table? (Thus enabling the use of `read` because it is required by default) 

@specify/ux-testing Feel free to provide feedback on this topic

Reply:
Hmm I highly doubt "read" or "view" is required in the first place. What use cases do you have in mind for adding a view permission? It matters even less since the uniqueness rules are fetched during context loading, so someone can see all uniqueness rules by just going to networks tab.

Reply:
That is, there is no need for supporting "view" or "read" permission action, since it. by implementation, is always allowed for anyone using specify ;)